tokio/src/runtime/blocking/pool.rs

```diff
@@ -84,6 +111,19 @@ where
     rt.spawn_blocking(func)
 }
 
+cfg_fs! {
+    #[cfg_attr(test, allow(dead_code))]
+    /// Runs the provided function on an executor dedicated to blocking operations.
```

Comment:
TODO: this comment was copy-pasted from `spawn_mandatory` and deserves an update